playbooks/roles/memcached/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: install memcached
+  apt: pkg=memcached state=present
```

Comment:
@mrgnr I don't have any strong opinions on edX's Ansible conventions (mostly because I very rarely write Ansible code), so your opinion definitely carries more weight here. But, if we start following our own conventions it makes it harder to push code upstream when we do have something to contribute. We'll needless need to refactor our code before we contribute it back, which makes us less likely to contribute in the first place. I'd really like us to get better at pushing code upstream because it benefits us when we no longer need to maintain it in our fork and it benefits the community/edX when they get some improved functionality. 

And I agree that handlers aren't an anti-pattern, but from the conference, one of the devops guys was telling me that there were certain conditions where the handlers weren't called even though a config file was changed. Sorry I don't have any more details. Maybe this is something we should bring up during the next ops hangout.